_posts/2019-06-19-load-image-data.md

```diff
@@ -0,0 +1,1095 @@
+---
+layout: post
+title: Load Large Image Data with Dask Array
+author: John Kirkham
+draft: true
+tags: [python,scipy,scikit-image,dask-image]
+theme: twitter
+---
+{% include JB/setup %}
+
+Executive Summary
+-----------------
+
+This post explores simple workflows to load large stacks of image data with Dask array.
+
+In particular, we start with a directory full of TIFF files of images like the
```

Comment:
perhaps link a folder with sample data? The folks following along can download the exact set of files from here
https://drive.google.com/open?id=12UE8ZNmcnDFxnhvx1WRc0tT3yBrsRjgr

Reply:
Whoa, Gokul joined GitHub :)  Welcome!

Reply:
I've gone and added this link in a couple places.  Thanks @rxist525